exchanges/ftx/ftx_wrapper.go

```diff
@@ -177,6 +177,7 @@ func (f *FTX) Setup(exch *config.ExchangeConfig) error {
 		GenerateSubscriptions:            f.GenerateDefaultSubscriptions,
 		Features:                         &f.Features.Supports.WebsocketCapabilities,
 		OrderbookBufferLimit:             exch.WebsocketOrderbookBufferLimit,
+		BufferEnabled:                    exch.WebsocketOrderbookBufferEnabled,
```

Comment:
FTX ORDERBOOKS R BROKENNNNN asks r completely empty.

Reply:
![image](https://user-images.githubusercontent.com/9391715/101446667-cee5f300-3977-11eb-8a23-852db368a624.png)

Seems to be broken on their end: https://ftx.com/api/markets/MSTR-1225/orderbook?depth=0
Have alerted their devs.